### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by kelvinqian00**

### CODE REVIEW

**Review by milt — COMMENTED** [review]
Forget ye not the `.gitignore`, ignore, ignore, the `.DS_Store` 🎵

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
 # action-deploy-clojars
```
Comment (milt) [inline README.md:1]: Is it worth mentioning in this doc that you'll need our setup-env or at least setup-clojure to use this?

Comment (milt) [reply README.md:1]: Or maybe just more generally that you'll need the clojure CLI on there

Comment (kelvinqian00) [reply README.md:1]: https://github.com/yetanalytics/action-deploy-clojars/pull/1/commits/dfe293df0d5e4c671e9e1b1a115e128e3cf674ec

**Review by milt — APPROVED** [review]
Looks great.

**Pull request merged**